src/redis_cmd.cc

```diff
@@ -242,7 +242,13 @@ class CommandFlushAll : public Commander {
 class CommandPing : public Commander {
  public:
   Status Execute(Server *svr, Connection *conn, std::string *output) override {
-    *output = Redis::SimpleString("PONG");
+    if (args_.size() == 1) {
+      *output = Redis::SimpleString("PONG");
+    } else if (args_.size() == 2) {
+      *output = Redis::SimpleString(args_[1]);
```

Comment:
IIRC Redis will return bulk string here. That is,

```bash
$ echo -e '*1\r\n$4\r\nPING\r\n' | nc localhost 6379
+PONG

$ echo -e '*2\r\n$4\r\nPING\r\n$4\r\nPONG\r\n' | nc localhost 6379
$4
PONG
```

Reply:
How do we determine SimpleString and BulkString in the go test case? @tisonkun

Reply:
Using `server.NewTCPClient` to obtain a low-level client to verify the raw response.

Reply:
Ok, thank you. That's exactly what I need :)